deploy/compose/container_files/nginx/.htpasswd

```diff
@@ -0,0 +1 @@
+admin:$apr1$XFCFfzu4$vtfox1qeBPqcZv814kltL1
```

Comment:
The next step is to make this configurable for the person deploying the application.

Reply:
Indeed, that comment ended up on the wrong file. I mean the one for the Helm chart. I'll add the comment there.